plugins/generator-1.19.4/forge-1.19.4/procedures/play_sound.java.ftl

```diff
@@ -0,0 +1,12 @@
+<#include "mcelements.ftl">
+if (world instanceof Level _level) {
+	if (!_level.isClientSide()) {
+		_level.playSound(null, ${toBlockPos(input$x,input$y,input$z)},
+	    	ForgeRegistries.SOUND_EVENTS.getValue(new ResourceLocation("${generator.map(field$sound, "sounds")?replace("CUSTOM:", "${modid}:")}")),

```

Comment:
It's because `Sounds.java` does the replacing in `getMappedValue()`. The sounds datalist doesn't have a `_mcreator_map_template` field like other lists